lib/workers/repository/updates/generate.ts

```diff
@@ -170,6 +170,7 @@ export function generateBranchConfig(
     // Use templates to generate strings
     if (upgrade.semanticCommits === 'enabled' && !upgrade.commitMessagePrefix) {
       logger.trace('Upgrade has semantic commits enabled');
+      upgrade.useSemanticCommit = true;
```

Comment:
is this a known defined property?

Reply:
Sorry, I'm unsure what a "known defined property" is, can you point me to a place where I can read up on it?